cmd/start-amazon-cloudwatch-agent/path.go

```diff
@@ -24,13 +24,12 @@ func startAgent(writer io.WriteCloser) error {
 	if os.Getenv(config.RUN_IN_CONTAINER) == config.RUN_IN_CONTAINER_TRUE {
 		// Use exec so PID 1 changes to agent from start-agent.
 		execArgs := []string{
-			agentBinaryPath, // when using syscall.Exec, must pass binary name as args[0]
-			"-config", tomlConfigPath,
-			"-envconfig", envConfigPath,
-			"-otelconfig", yamlConfigPath,
+			paths.AgentBinaryPath, // when using syscall.Exec, must pass binary name as args[0]
+			"-config", paths.TomlConfigPath,
+			"-envconfig", paths.EnvConfigPath,
```

Comment:
I removed passing the `-otelconfig` option since it is not needed. Then again, I could put it back in.

Anyone care either way?

Reply:
I'd rather keep it, but it's not a big deal.

Reply:
I'll re add it.